app/api-v1/routes/order.js

```diff
@@ -1,11 +1,19 @@
 // eslint-disable-next-line no-unused-vars
-module.exports = function (orderService) {
+const { BadRequestError } = require('../../utils/errors')
+module.exports = function (orderService, identityService) {
   const doc = {
     GET: async function (req, res) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      const { address: supplierAddress } = await identityService.getMemberByAlias(req, req.body.supplier)
+
+      const { statusCode, result } = await orderService.postOrder({ ...req.body, supplier: supplierAddress })
+      if (!req.body) {
+        throw new BadRequestError({ message: 'No body uploaded', service: 'order' })
+      }
+      result[0].supplier = req.body.supplier
+      return res.status(statusCode).json(result)
```

Comment:
This should be returning an Object not an array if you check the OpenAPI spec. I'd also prefer we avoid the mutation you have on the line above; maybe something like:

```js
return res.status(statusCode).json({
   ...result[0],
   supplier: req.body.supplier
})
```

You may also need to watch out for extra properties from the database if they exist